### PR TITLE
Add boardsource/sessanta

### PR DESCRIPTION
**Pull request by waffle87**
## Description

Adds v3 definition of Boardsouce Sessanta keyboard

## QMK Pull Request

[qmk/qmk_firmware:23823](https://github.com/qmk/qmk_firmware/pull/23823)

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have formatted the JSON file to have consistent formatting with the rest of the repository.
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/boardsource/sessanta/sessanta.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Sessanta",
+    "vendorId": "0x4273",
+    "productId": "0x17AC",
+    "matrix": {"rows": 8, "cols": 9},
+    "keycodes": ["qmk_lighting"],
+    "menus": ["qmk_rgb_matrix"],
+    "layouts": {
+        "labesl": [["Bottom Row", "ANSI", "Split"]],
```
Comment (Xelus22) [inline v3/boardsource/sessanta/sessanta.json:9]:
```suggestion
        "labels": [["Bottom Row", "ANSI", "Split"]],
```
typo

Comment (waffle87) [reply v3/boardsource/sessanta/sessanta.json:9]: Thank you :)

File: v3/boardsource/sessanta/sessanta.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        "keymap": [
+            ["0,0","0,1","0,2","0,3","0,4","0,5","0,6","0,7","0,8","1,0","1,1","1,2","1,3",{"w": 2},"1,4"],
+            [{"w": 1.5},"1,5","1,6","1,7","1,8","2,0","2,1","2,2","2,3","2,4","2,5","2,6","2,7","2,8",{"w": 1.5},"3,0"],
+            [{"w": 1.75},"3,1","3,2","3,3","3,4","3,5","3,6","3,7","3,8","4,0","4,1","4,2","4,3",{"w": 2.25},"4,4","4,5"],
+            [{"w": 2.25},"4,6","4,7","4,8","5,0","5,1","5,2","5,3","5,4","5,5","5,6","5,7",{"w": 2.75},"5,8","6,0"],
+            [{"w": 1.25},"6,1",{"w": 1.25},"6,2",{"w": 1.25},"6,3",{"w": 6.25},"6,4\n\n\n0,0",{"w": 1.25},"6,5",{"w": 1.25},"6,6",{"w": 1.25},"6,7",{"w": 1.25},"6,8","7,0"],
+            [{"x": 3.75,"w": 1.5},"7,1\n\n\n0,1","7,2\n\n\n0,1",{"w": 1.25},"6,4\n\n\n0,1","7,3\n\n\n0,1",{"w": 1.5},"7,4\n\n\n0,1"]
+        ]
+    }
+}
```
Comment (Cipulot) [inline v3/boardsource/sessanta/sessanta.json:19]:
```suggestion
{
  "name": "Sessanta",
  "vendorId": "0x4273",
  "productId": "0x17AC",
  "matrix": {
    "rows": 8,
    "cols": 9
  },
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_rgb_matrix"
  ],
  "layouts": {
    "labels": [
      [
        "Bottom Row",
        "ANSI",
        "Split"
      ]
    ],
    "keymap": [
      [
        {
          "c": "#aaaaaa"
        },
        "0,0",
        {
          "c": "#cccccc"
        },
        "0,1",
        "0,2",
        "0,3",
        "0,4",
        "0,5",
        "0,6",
        "0,7",
        "0,8",
        "1,0",
        "1,1",
        "1,2",
        "1,3",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "1,4"
      ],
      [
        {
          "w": 1.5
        },
        "1,5",
        {
          "c": "#cccccc"
        },
        "1,6",
        "1,7",
        "1,8",
        "2,0",
        "2,1",
        "2,2",
        "2,3",
        "2,4",
        "2,5",
        "2,6",
        "2,7",
        "2,8",
        {
          "w": 1.5
        },
        "3,0"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "3,1",
        {
          "c": "#cccccc"
        },
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        "3,7",
        "3,8",
        "4,0",
        "4,1",
        "4,2",
        "4,3",
        {
          "c": "#aaaaaa",
          "w": 2.25
        },
        "4,4",
        "4,5"
      ],
      [
        {
          "w": 2.25
        },
        "4,6",
        {
          "c": "#cccccc"
        },
        "4,7",
        "4,8",
        "5,0",
        "5,1",
        "5,2",
        "5,3",
        "5,4",
        "5,5",
        "5,6",
        "5,7",
        {
          "c": "#aaaaaa",
          "w": 2.75
        },
        "5,8",
        "6,0"
      ],
      [
        {
          "w": 1.25
        },
        "6,1",
        {
          "w": 1.25
        },
        "6,2",
        {
          "w": 1.25
        },
        "6,3",
        {
          "c": "#cccccc",
          "w": 6.25
        },
        "6,4\n\n\n0,0",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "6,5",
        {
          "w": 1.25
        },
        "6,6",
        {
          "w": 1.25
        },
        "6,7",
        {
          "w": 1.25
        },
        "6,8",
        "7,0"
      ],
      [
        {
          "x": 3.75,
          "c": "#cccccc",
          "w": 1.5
        },
        "7,1\n\n\n0,1",
        "7,2\n\n\n0,1",
        {
          "w": 1.25
        },
        "6,4\n\n\n0,1",
        "7,3\n\n\n0,1",
        {
          "w": 1.5
        },
        "7,4\n\n\n0,1"
      ]
    ]
  }
}

```

**Review by Cipulot — APPROVED** [review]

**Comment by Cipulot** [comment]
VIA keymap moved after last QMK BC here: https://github.com/the-via/qmk_userspace_via/tree/main/keyboards/boardsource/sessanta/keymaps/via

**Pull request merged**